src/LndClient/Data/BtcEnv.hs

```diff
@@ -0,0 +1,22 @@
+module LndClient.Data.BtcEnv
+  ( BtcEnv (..),
+    btcEnv,
+  )
+where
+
+import Env hiding (def)
+
+data BtcEnv
+  = BtcEnv
+      { btcRpcUser :: String,
+        btcRpcPassword :: String,
+        btcRpcUrl :: String
```

Comment:
btw you can use `ByteString` for these types (to avoid `Char8.pack` later)

Reply:
probably except `btcRpcUrl`, you are using it as string

Reply:
Okay, done, works)